cmd/cli/plugin/secret/registry_secret_add.go

```diff
@@ -74,16 +84,40 @@ func registrySecretAdd(cmd *cobra.Command, args []string) error {
 			}
 		}
 		log.Info("\n")
-	}
-
-	pkgClient, err := tkgpackageclient.NewTKGPackageClient(registrySecretOp.KubeConfig)
-	if err != nil {
-		return err
-	}
+	} else {
+		secretExportList, err := pkgClient.ListSecretExports(registrySecretOp)
```

Comment:
Could we get the SecretExport with the name = registrySecretOp.SecretName instead of listing all? Would remove the need to do a for loop on line 93.

Reply:
Fixed